Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:20.15.1
```

Comment:
Could this run on alpine?

Reply:
I'm not sure as I didn't test with Alpine. My goal was to match the requirements laid out in the markdown files for what the team says it takes to run the app.